backend/controllers/homepage/homepage.ts

```diff
@@ -190,7 +190,10 @@ const getTasks = ({ activeCaseLoadId, locations, staffId, whereaboutsConfig, key
       heading: 'Manage prison visits',
       description: 'Book, view and cancel a prisoner’s social visits.',
       href: managePrisonVisits.ui_url,
-      enabled: () => managePrisonVisits.ui_url && userHasRoles(['MANAGE_PRISON_VISITS']),
+      enabled: () =>
+        managePrisonVisits.ui_url &&
+        userHasRoles(['MANAGE_PRISON_VISITS']) &&
+        managePrisonVisits.enabled_prisons.split(',').includes(activeCaseLoadId),
```

Comment:
would be good to add a test for this change

Reply:
Looking now